stellar/contracts/wormhole-contract/src/vaa/mod.rs

```diff
@@ -0,0 +1,216 @@
+mod signature;
+
+pub use signature::Signature;
+
+use crate::utils::BytesReader;
+use core::convert::TryFrom;
+use soroban_sdk::{contracttype, Bytes, BytesN, Env, Vec};
+use wormhole_interface::Error;
+
+#[contracttype]
+#[derive(Clone, Debug, PartialEq)]
+pub struct VAA {
```

Comment:
Why are we defining `VAA` again here when it has already been defined in the interface?

Reply:
Correct

Reply:
We decided to move to a new PR.